src/cardslide.js

```diff
@@ -1,25 +1,55 @@
 import React, {Component} from 'react'
-import SlideCard from './slidecard'
 
 export default class CardSlide extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isClicked: false
+    }
+    this.handleClick = this.handleClick.bind(this)
+    this.handleOnClick = this.handleOnClick.bind(this)
+  }
+
+  handleOnClick() {
+    this.setState((state) => ({
+      isClicked: !state.isClicked
+    }))
+  }
 
   handleClick(direction) {
     this.props.onClick(direction)
+    this.setState({
+      isClicked: false
+    })
   }
+
   render() {
-    const {translateValue, cardInfo} = this.props
-    const cards = cardInfo.map((card, i) => (
-      <SlideCard card = {card} key={i}/>
-    ))
+    const {question, answer} = this.props.cardInfo
+    const {cardCount, page} = this.props
+    const {isClicked} = this.state
+    const progressWidth = ((page + 1) === cardCount) ? 100 : parseInt((100 / cardCount), 10) * (page + 1)
+    const styleArrow = {
```

Comment:
For static inline styles, it's better to define them in the global scope (`styles` object) of the module instead of in a function or render method.